File: EB/01.21/1924.py
# 1924. 2007년

month = [31, 28, 31, 30, 31, 30, 31, 31, 30, 31, 30, 31]
week = ["SUN", "MON", "TUE", "WED", "THU", "FRI", "SAT"]
day = 0

x, y = map(int, input().split())

for i in range(x-1):
    day += month[i]

day = (day + y) % 7
print(week[day])

File: Lami/01.30/1924.js
const fs = require('fs');
const filePath = process.platform === 'linux' ? '/dev/stdin' : './input.txt';
let input = fs.readFileSync(filePath).toString().split('\n');

function solution() {
  const month = Number(input[0].split(' ')[0]);
  const day = Number(input[0].split(' ')[1]);
  const days = ['MON', 'TUE', 'WED', 'THU', 'FRI', 'SAT', 'SUN'];
  const monthToDays = {
    28: [2],
    30: [4, 6, 9, 11],
    31: [1, 3, 5, 7, 8, 10, 12],
  };
  let sum = 0;

  for (let i = 1; i < month; i++) {
    for (let m in monthToDays) {
      if (monthToDays[m].includes(i)) {
        sum += Number(m);
      }
    }
  }
  sum += day - 1;
  console.log(days[sum % 7]);
}

solution();
